### ISSUE TITLE
Allow for using different output devices for ringing and voice

### ISSUE THREAD

**Issue by gecame** [cern-phone-apps/desktop-phone-app#220]
**Is your feature request related to a problem? Please describe.**
Currently, the 'Devices' setting allows for selecting a generic "Output" device. This device is then used for generating the ring tone as well as for RTP (voice) transmission. However, it would be advantageous to separate the two and to allow separate configurations.

Consider the following use case: A user has a headset connected to his PC which he uses for telephony exclusively. During an incoming call, he wants to hear the ringtone over the standard PC loudspeaker. Only then he will put on his headphones and accept the call, with voice being transmitted over the headset.

**Describe the solution you'd like**
In addition to "Output device" (that could be renamed to "Voice output device"), provide an additional pull-down menu for "Ringing device" that allows selecting a different device for ringing than the one used for voice.

**Comment by renefs** [comment]
Hi, this is one feature that is pending on the PR's queue: #196

**Comment by gecame** [comment]
Excellent - please feel free to mark it as a duplicate and close it!
Apologies and cheers, Germán

**Comment by renefs** [comment]
Duplicate of #110
